src/meta/client/MetaClient.cpp

```diff
@@ -1156,11 +1160,25 @@ folly::Future<StatusOr<bool>> MetaClient::heartbeat() {
     return future;
 }
 
-folly::Future<StatusOr<int64_t>> MetaClient::balance(bool isStop) {
+folly::Future<StatusOr<int64_t>> MetaClient::balance(std::vector<HostAddr> hostDel,
```

Comment:
because it never stolen the memory of hostDel in the function body, so maybe std::vector<HostAddr> &hostDel is better, this will not call right copy function.